src/sinks/util/rusoto.rs

```diff
@@ -4,11 +4,20 @@ use hyper_openssl::{
     openssl::ssl::{SslConnector, SslMethod},
     HttpsConnector,
 };
-use rusoto_core::HttpClient;
+use rusoto_core::{CredentialsError, HttpClient, Region};
+use rusoto_credential::{DefaultCredentialsProvider, ProvideAwsCredentials};
+use rusoto_sts::{StsAssumeRoleSessionCredentialsProvider, StsClient};
+use snafu::{ResultExt, Snafu};
 
 pub type Client = HttpClient<HttpsConnector<HttpConnector<Resolver>>>;
 
-pub fn client(resolver: Resolver) -> crate::Result<Client> {
+#[derive(Debug, Snafu)]
+enum RusotoError {
+    #[snafu(display("{}", source))]
```

Comment:
```suggestion
    #[snafu(display("Invalid AWS credentials: {}", source))]
```

Do we want to add some sort of context here?

Reply:
That was cloned from the `aws_cloudwatch_logs` sink. I agree it should have more context. I'll add it.